src/client/src/orangehrmRecruitmentPlugin/pages/ShortlistCandidateScreen.vue

```diff
@@ -0,0 +1,182 @@
+              <oxd-input-field
+                v-model="shortlist.note"
+                label="notes"
+                type="textarea"
+                :rules="rules.note"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+        <oxd-form-actions>
+          <required-text />
+          <oxd-button display-type="ghost" :label="$t('general.back')" />
+          <submit-button label="Shortlist" />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {shouldNotLessThanCharLength} from '@ohrm/core/util/validation/rules';
+import {APIService} from '@/core/util/services/api.service';
+import {navigate} from '@/core/util/helper/navigation';
+export default {
+  name: 'ShortlistCandidateScreen',
+  setup() {
+    const http = new APIService(
+      'https://01eefc6d-daf1-4643-97ae-2d15ea8b587b.mock.pstmn.io',
+      'recruitment/shortlistCandidate',
+    );
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      isLoading: false,
+      rules: {
+        notes: [shouldNotLessThanCharLength(250)],
+      },
+      shortlist: {
+        candidate: '',
+        vacancy: '',
+        manager: '',
+        status: '',
+        note: '',
+      },
+      statuses: [
+        {
+          id: 1,
+          label: this.$t('recruitment.application_initiated'),
+        },
+        {
+          id: 2,
+          label: this.$t('recruitment.shortlisted'),
+        },
+        {
+          id: 3,
+          label: this.$t('recruitment.interview_scheduled'),
+        },
+        {
+          id: 4,
+          label: this.$t('recruitment.interview_passed'),
+        },
+        {
+          id: 5,
+          label: this.$t('recruitment.interview_failed'),
+        },
+        {
+          id: 6,
+          label: this.$t('recruitment.job_offered'),
+        },
+        {
+          id: 7,
+          label: this.$t('recruitment.offered_declined'),
+        },
+      ],
+    };
+  },
+  beforeMount() {
+    this.isLoading = true;
+    this.http.getAll().then(({data: {data}}) => {
+      const {candidate, status, ...rest} = data;
+      const {firstName, lastName, middleName} = candidate;
+      const fullName = `${firstName} ${middleName} ${lastName}`;
+      this.shortlist = {
+        candidate: fullName,
+        status: this.statuses.find(({id}) => id === status)?.label,
+        ...rest,
+      };
+      this.isLoading = false;
+    });
+  },
+  methods: {
+    onSave() {
+      this.isLoading = true;
+      this.http
+        .update(this.shortlist.id, {note: this.shortlist.note})
+        .then(() => {
+          return this.$toast.updateSuccess();
+        })
+        .then(() => {
+          navigate('/recruitment/viewCandidates');
+        });
+    },
+  },
+};
+</script>
+<style scoped lang="scss">
+.orangehrm-save-candidate-page {
+  &-full-width {
+    grid-column: 1 / span 2;
```

Comment:
why does this needed?

Reply:
text box has capturing two columns

Reply:
use class `--span-column-2` it's globally available